examples/flavor/swarm/flavor.go

```diff
@@ -228,6 +257,10 @@ func (s *baseFlavor) prepare(role string, flavorProperties *types.Any, instanceS
 			log.Warningln("Cannot prepare:", err)
 		}
 
+		if dockerClient != nil {
+			dockerClient.(io.Closer).Close()
```

Comment:
docker client ensures that transport.Client is closed before close ( https://github.com/docker/docker/blob/master/client/client.go#L183 ).
I am worried about io.Closer wouldn't care that. 
It is more safe to do same way as docker client.